Assets/Resources/BillingMode.json

```diff
@@ -0,0 +1 @@
+{"androidStore":"GooglePlay"}
```

Comment:
What does `Assets/Resources/BillingMode.json` do? We need to commit it to the repo or should it be added to `.gitignore`?